cmd/ctlog/managefulcio/main.go

```diff
@@ -0,0 +1,173 @@
+// Copyright 2022 The Sigstore Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"net/url"
+	"os"
+	"strings"
+
+	fulcioclient "github.com/sigstore/fulcio/pkg/api"
+	"github.com/sigstore/scaffolding/pkg/ctlog"
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/types/known/anypb"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"knative.dev/pkg/logging"
+	"knative.dev/pkg/signals"
+	"sigs.k8s.io/release-utils/version"
+)
+
+const (
+	// Key in the configmap holding the value of the tree.
+	treeKey   = "treeID"
+	configKey = "config"
+	bitSize   = 4096
+)
+
+var (
+	cmname           = flag.String("configmap", "ctlog-config", "Name of the configmap where the treeID lives. if configInSecret is false, ctlog config gets added here also.")
+	configInSecret   = flag.Bool("config-in-secret", false, "If set to true, fetch / update the ctlog configuration proto into a secret specified in ctlog-secrets under key 'config'.")
+	secretName       = flag.String("secret", "ctlog-secrets", "Name of the secret to fetch private key for CTLog.")
+	pubKeySecretName = flag.String("pubkeysecret", "ctlog-public-key", "Name of the secret to fetch public key from.")
+	fulcioURL        = flag.String("fulcio-url", "http://fulcio.fulcio-system.svc", "Where to fetch the fulcio Root CA from.")
+	operation        = flag.String("operation", "", "Operation to perform for the specified fulcio [add,remove]")
```

Comment:
I think this could also be used for non-fulcio certificates too - let's say we add in third-party CA certificates. can we update the documentation and name of the job (managectroots?) for this? might want to update `fulcioOp` too